app/locales/en/translations.ts

```diff
@@ -532,6 +532,7 @@ export default {
         min_subjects: 'You must select at least {{minLength}} subject(s).',
         node_license_invalid: 'Invalid required fields for the license',
         node_license_missing_fields: 'The following required fields are missing: {{missingFields}}',
+        invalid_doi: 'Please use a valid DOI format (10.xxx/xxxxx)',
```

Comment:
The format here should match the placeholder (there must be at least 4 numbers after `10.`). Either update this to match or (if it's easy, better to) pass in the placeholder so it will always match.